<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CSdrSetupDlg</class>
 <widget class="QDialog" name="CSdrSetupDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>356</width>
    <height>202</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SDR Setup</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>150</y>
     <width>191</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBoxRate">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>201</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>Bandwidth / Sample Rate</string>
   </property>
   <widget class="QRadioButton" name="radioButtonRate2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>171</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>500 KHz</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButtonRate1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>171</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>100 KHz</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButtonRate0">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>171</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>50 KHz </string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButtonRate3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>171</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>2000 KHz</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBoxGain">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>10</y>
     <width>120</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>RF Gain/Attn</string>
   </property>
   <widget class="QRadioButton" name="radioButtonAttn0">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>0 dB</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButtonAttn10">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>-10 dB</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButtonAttn20">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>-20 dB</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButtonAttn30">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>-30 dB</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="labelRadioType">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>10</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string>Radio Type</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>CSdrSetupDlg</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>201</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>CSdrSetupDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>201</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButtonAttn10</sender>
   <signal>clicked()</signal>
   <receiver>CSdrSetupDlg</receiver>
   <slot>RfGainChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>281</x>
     <y>61</y>
    </hint>
    <hint type="destinationlabel">
     <x>215</x>
     <y>123</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButtonAttn20</sender>
   <signal>clicked()</signal>
   <receiver>CSdrSetupDlg</receiver>
   <slot>RfGainChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>247</x>
     <y>81</y>
    </hint>
    <hint type="destinationlabel">
     <x>230</x>
     <y>129</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButtonAttn30</sender>
   <signal>clicked()</signal>
   <receiver>CSdrSetupDlg</receiver>
   <slot>RfGainChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>294</x>
     <y>96</y>
    </hint>
    <hint type="destinationlabel">
     <x>303</x>
     <y>127</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButtonAttn0</sender>
   <signal>clicked()</signal>
   <receiver>CSdrSetupDlg</receiver>
   <slot>RfGainChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>300</x>
     <y>36</y>
    </hint>
    <hint type="destinationlabel">
     <x>346</x>
     <y>53</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>SelectNetSDR()</slot>
  <slot>SelectSDRIQ()</slot>
  <slot>RfGainChanged()</slot>
 </slots>
</ui>
